Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -632,7 +632,7 @@ public List<int> Liquidate(Symbol symbolToLiquidate = null)
                     }
                     else
                     {
-                        Transactions.CancelOrder(order.Id);
+                        Transactions.CancelOrder(order.Id,"Liquidated");
```

Comment:
Perhaps we could accept the string as an argument with a default value of liquidated. This would be consistent with the other methods that create orders and accept a tag.